### PR TITLE
PCC ntupler 76x - October2015

### PR DESCRIPTION
**Pull request by capalmer85**
update with BX splitting option; 
includeJets (default off); 
add .py for data certification with express stream

### CODE REVIEW

**Comment by capalmer85** [comment]
@davidlange6 @slava77  I'd like to get this merged relatively quickly because there are some significant updates that the lumi data certifiers need access to on a short time scale.

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc [hunk 1]
```diff
@@ -114,6 +116,28 @@ PCCNTupler::PCCNTupler(edm::ParameterSet const& iConfig):
         pileUpToken=consumes<std::vector< PileupSummaryInfo> >(fPileUpInfoLabel);
         tree->Branch("nPU",&nPU,"nPU/I");
     }
+
+    if(includeJets){
+        hltjetsToken_=consumes<reco::CaloJetCollection >(edm::InputTag("ak4CaloJets"));
+        const int kMaxJetCal = 10000;
```
Comment (slava77) [inline RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc]: 10k ak4 jets can't fit into CMS acceptance.
This looks excessive regarding memory.

Comment (capalmer85) [inline RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc]: (I stole this from elsewhere in cmssw, so I didn't look too carefully.)

What value would be acceptable?

Comment (slava77) [inline RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc]: even 100 is plenty enough.

Comment (capalmer85) [inline RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc]: Fixed.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by capalmer85** [comment]
@slava77 when should this be merged?

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/11832	fa1762c
- PCC analysis  ntupler update, not a part of centrally running workflows
- jenkins tests pass (compiles)

**Comment by davidlange6** [comment]
+1

**Pull request merged**